SerialPrograms/Source/PokemonSwSh/Inference/Sounds/PokemonSwSh_BerryTreeRustlingSoundDetector.cpp

```diff
@@ -0,0 +1,48 @@
+﻿/*  Berry Tree Rustling Sound Detector
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#include <QString>
```

Comment:
minor but I think some header could be cleaned up !

Reply:
Yeah, we're in the process of removing Qt from anything that isn't UI code. So you shouldn't need `QString` unless you need some of its library functions which are available elsewhere..

Reply:
fixed